apps/OboeTester/app/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.4.1)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Werror -Wall -std=c++14")
```

Comment:
consider adding -Wextra

Reply:
@krocard - yes, this is a huge checkin. I have been developing it for several years in "vendor/box". It was the experimental platform that led to AAudio. When I extracted AAudio then it because a handy test tool. Now I just need to put it where SOC partners can get the code. They are already using the binaries.

I added you as a reviewer because you recommended using unique_ptr in "NativeAudioContext.h".

I will make the cleanup changes you recommend. Thanks. But no need to review the whole thing. I will be doing general cleanup and refactoring over time.

Reply:
-Wextra caused too many warnings. Will do in a subsequent patch so we have a history.